compose/snippets/src/main/java/com/example/compose/snippets/LandingScreen/LandingScreen.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.compose.snippets.LandingScreen
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.compose.snippets.navigation.Destination
+
+@Composable
+fun LandingScreen(
+    toBrushExamples: () -> Unit,
+    toImageExamples: () -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .padding(16.dp)
+            .fillMaxSize(),
+        verticalArrangement = Arrangement.spacedBy(24.dp),
+    ) {
+        Text(
+            modifier = Modifier.fillMaxWidth(),
+            style = TextStyle(
+                fontSize = 24.sp,
+                fontWeight = FontWeight.Bold,
+                textAlign = TextAlign.Center,
+            ),
+            text = "Android snippets",
+        )
+        Text(
+            text = "Use the following buttons to view a selection of the snippets used in the Android documentation."
+        )
+        NavigationButtons(toBrushExamples, toImageExamples)
+    }
+}
+
+@Composable
+fun NavigationButtons(toBrushExamples: () -> Unit, toImageExamples: () -> Unit) {
+    LazyColumn(
+        modifier = Modifier
+            .padding(16.dp)
+            .fillMaxSize(),
+        verticalArrangement = Arrangement.spacedBy(8.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        items(Destination.values().toList()) { destination ->
+            when (destination) {
```

Comment:
As well as this `when` would be simplified to just one `NavigationButton()` (i think this should also be called `NavigationItem`), that just passes the destination to the callback.

Reply:
I'm not sure if I've understood correctly what you mean by having just one `NavigationItem`. How does it look now?

Reply:
You should be able to remove the `when` and just have 
```
NavigationItem(destination) {
                    navigate(
                        destination
                    )
                }
```
Since the branches of the when are doing the same thing.

Reply:
Got it! Nice spot. Pushed in [this commit](https://github.com/android/snippets/pull/130/commits/de74866cf5b11716d8e1102f36e03b5e07f54cc0).